plugins/reporters/cyclonedx/src/main/kotlin/CycloneDxReporter.kt

```diff
@@ -274,9 +274,8 @@ class CycloneDxReporter : Reporter {
                             severity = org.cyclonedx.model.vulnerability.Vulnerability.Rating.Severity
                                 .fromString(reference.severityRating.lowercase())
                             score = reference.severity?.toDoubleOrNull()
-                            method = org.cyclonedx.model.vulnerability.Vulnerability.Rating.Method.fromString(
-                                reference.scoringSystem
-                            )
+                            method = org.cyclonedx.model.vulnerability.Vulnerability.Rating.Method
+                                .fromString(reference.scoringSystem)
```

Comment:
As the "broken" formatting was introduced in this PR, please just squash this into the previous commit instead of creating a new one. generally, we prefer review comments to be addressed in the commit they belong to, and force-push after amending / fixing up commits.

Reply:
Sure, thanks for letting me know!